bot/exts/utils/clean.py

```diff
@@ -89,6 +163,38 @@ def predicate_regex(message: Message) -> bool:
             await ctx.send(embed=embed)
             return
 
+        if after_message:
+
+            # Ensure that until_message is specified.
+            if not until_message:
+                embed = Embed(
+                    color=Colour(Colours.soft_red),
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description="`until_message` must be specified if `after_message` is specified."
+                )
+                await ctx.send(embed=embed)
+                return
+
+            # Check if the messages are not in same channel
+            if after_message.channel != until_message.channel:
+                embed = Embed(
+                    color=Colour(Colours.soft_red),
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description="You cannot do range clean across different channel."
```

Comment:
```suggestion
                    description="You cannot do range clean across several channels."
```

Reply:
Sounds better, alias and this was changed in https://github.com/python-discord/bot/pull/1527/commits/c2705492ec13984aa75f4c31532ca18b1756121d, thanks